typescript/sdk/src/providers/ProviderType.ts

```diff
@@ -21,10 +21,12 @@ import type {
   Transaction as VTransaction,
   TransactionReceipt as VTransactionReceipt,
 } from 'viem';
+import * as zk from 'zksync-ethers';
```

Comment:
do we need everything? would be nice to have it similar to the existing format by only importing the relevant items and adding the `zk` prefix

e.g
- `zk.Contract` ends up being `ZKSyncContract` or something to that effect
- `zk.types.TransactionRequest` -> `ZKSyncTransactionRequest`